README.md

```diff
@@ -110,6 +87,47 @@ if(response.reason == ImageStudioReason.CLOSED) {
 }
 ```
 
+
+## AmplienceImageStudioOptions
+
+When creating an `AmplienceImageStudio` instance, the constructor takes an `AmplienceImageStudioOptions` object. This defines several options for customizing the behaviour of the studio:
+
+| Key | Description | Default | Optional |
+|:----------|:-------------|:-:|:-:|
+| domain |  base domain for your image-studio deployment | - | No |
+| sdkMetadataOverride |    object containing behavioural overrides for image-studio [SDKMetadata](#sdkmetadata)   | {} | Yes |
+| windowTarget | window.open() `target` override, please see [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/open) | '_blank' | Yes |
+| windowFeatures | window.open() `windowFeatures` override, please see [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/open) | '' | Yes |
+
+### SDKMetadata
+
+This SDK controls certain behaviours within image-studio through optional enablement flags.
+
+_Failing to submit an option to image-studio will not result in any bad behaviour. Image-Studio determines and controls the default behaviour of every option, however the structure and options available are controlled by this SDK._
+
+| Option | Description | Default |
+|:----------|:-|:-:|
+| `allowImageSave` | allows content to be saved back to the SDK | `false` |
+| `allowLogout` | allows users to logout | `true` |
+| `allowCreate` | allows users to create new content | `true` |
+|`orgId`*| user organisation ID (used for entitlements and credit consumption) | ''
```

Comment:
Missing a | at the end of the table?

Reply:
will do a docs update in another branch to not hold this up any longer!